flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -96,6 +91,21 @@
 
 	protected ResultPartitionID[] partitionIds;
 
+	public void setUp(
```

Comment:
`@Depreacted`?

Reply:
not really...if you don't need a custom `Configuration` object, you can still use this as a shortcut(?)

Reply:
as you prefer